API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
++ `access_token_endpoint` is the token endpoint used by client to obtain the access token. Value type is **string**. "/access_token" is the **default**.
++ `revocation_endpoint` is the revocation endpoint used by client to revoke a token. Value type is **string**. "/revoke" is the **default**.
++ `token_header_keyword` represents the string used instead of "Bearer" in "Authorization" header field according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.1). Value type is **string**. "Bearer" is the **default**.
++ `token_body_keyword` represents the parameter name used instead of "access_token" in the body of an API request according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.2). Value type is **string**. "access_token" is the **default**.
++ `token_query_keyword` represents the parameter name used instead of "access_token" in the query of an API request according to [RFC 6750](http://tools.ietf.org/html/rfc6750#section-2.3). Value type is **string**. "access_token" is the **default**.
++ `example_token` represents an example access token value. Value type is **string**. "example_token" is the **default**.
+
+> **NOTE:** The value for `oauth_resource_endpoint` can include hostnames. `authorization_endpoint` and `access_token_endpoint` allow it too only if `oauth_resource_endpoint` is not specified.
+
+##### Example
+
+    + oauth (OAuth2 Authentication Scheme)
+        + oauth_resource_endpoint: `/oauth`
+        + access_token_endpoint: `/token`
+        + token_header_keyword: `token`
+        + scopes: read, write, email
+
+#### Bearer Authentication Scheme
+This scheme represents the method of sending a bearer token as specified in [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1).
+
+The customizable options for this scheme are:
+
++ `keyword` represents the string used instead of "Bearer" in the "Authorization" header field according to [RFC 6750](https://tools.ietf.org/html/rfc6750#section-2.1). Value type is **string**. "Bearer" is the **default**.
++ `example_token` represents an example bearer token value. Value type is **string**. "example_token" is the **default**.
+
+##### Example
+
+    + token (Bearer Authentication Scheme)
+        + keyword: `token`
+
+---
+
+## Authenticated section
+- **Parent sections:** [Resource section](#def-resource-section), [Action section](#def-action-section), [Request section](#def-request-section)
+- **Nested sections:** none
+- **Markdown entity:** list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authenticated` keyword in Markdown list entity followed by optional comma seperated list of scheme names which are prefixed by a colon.
+
+    + Authenticated: <list>
+
+> **NOTE:** If name of an authentication scheme dervied from `OAuth2 Authentication Scheme` is used, it can optionally have comma seperated list of oauth scopes inside `[]` immediately following it.
```

Comment:
:+1: to @danielgtaylor's comment. I vote for example and explanation of exact interpretation of how scopes are handled.

btw

```diff
- oauth scopes
+ OAuth scopes
```